pkg/apis/machine/v1beta1/machine_webhook_test.go

```diff
@@ -1040,6 +1041,10 @@ func TestDefaultAWSProviderSpec(t *testing.T) {
 
 	clusterID := "clusterID"
 	region := "region"
+	arch := defaultAWSX86InstanceType
+	if runtime.GOARCH == "arm64" {
+		arch = defaultAWSARMInstanceType
+	}
```

Comment:
Do we run these tests on arm? I think it would be better if we could explicitly set up a test where we create the AWS defaulter with the arm architecture? Eg, could we pass the architecture into `createMachineDefaulter` to allow it to be specced from the test suite?

Reply:
Per-repo E2E tests are not being run on ARM at this time.